### ISSUE TITLE
Multiple applications to an object with 1 free attribute

### ISSUE THREAD

**Issue by Sitiritis** [cqfn/eo#286]
# Steps to reproduce

1. Place the following code in `sandbox/eo/app.eo`:

<details>
<summary>app.eo</summary>

```
+package sandbox
+alias stdout org.eolang.io.stdout
+alias sprintf org.eolang.txt.sprintf

[x] > empty
  x > @

[args...] > app
  (empty 5) > res1
  (res1 6) > res2
  (1 7) > res3
  seq > @
    stdout
      sprintf "res1 = %d\n" res1
    stdout
      sprintf "res2 = %d\n" res2
    stdout
      sprintf "res3 = %d\n" res3
```
</details>

2. Compile it
3. Run the compiled program

# Expected output

Compilation or runtime error, because `res1` does not have free attributes.

# Actual output

```
res1 = 5
res2 = 6
res3 = 1
```

`res1`) Ok
`res2`) Certainly not ok
`res3`) Ok, this can be interpreted by a programmer as data has free attributes, which is not used by the object, but is it okay from language design point of view?

**Comment by yegor256** [comment]
@Sitiritis the bug is still valid?

**Comment by Sitiritis** [comment]
Yes, the output is the same

**Comment by yegor256** [comment]
@rultor release, tag is `0.1.32`

**Comment by yegor256** [comment]
@Sitiritis I believe it's fixed, please check version `0.1.32`.

**Comment by Sitiritis** [comment]
@yegor256 everything is fine now, thank you!

<details>
<summary>Program output</summary>

res1 = 5
Exception in thread "main" java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: org.eolang.phi.Attr$Exception: Error at sandbox.EOempty#x in sandbox.EOempty#193064360:{
  _order=[x]
  x(0)=𝜂SFN
  ρ=sandbox.EOapp#812265671:{
    _order=[args, res1, res2, res3]
    args(0)=[]VN
    ρ=𝜂SN
    res1(1)=λOBN
    res3(3)=λOBN
    φ=λOBN
    res2(2)=λOBN
  }SN
  φ=λOBN
}
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:84)
        at org.eolang.phi.AtLambda.get(AtLambda.java:63)
        at org.eolang.phi.AtBound.get(AtBound.java:56)
        at org.eolang.phi.AtNamed.get(AtNamed.java:66)
        at org.eolang.phi.PhDefault.attr(PhDefault.java:143)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhDefault.attr(PhDefault.java:143)
        at org.eolang.phi.Dataized.take(Dataized.java:54)
        at org.eolang.phi.Dataized.take(Dataized.java:72)
        at org.eolang.phi.Main.exec(Main.java:80)
        at org.eolang.phi.Main.main(Main.java:57)
Caused by: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: org.eolang.phi.Attr$Exception: Error at sandbox.EOempty#x in sandbox.EOempty#193064360:{
  _order=[x]
  x(0)=𝜂SFN
  ρ=sandbox.EOapp#812265671:{
    _order=[args, res1, res2, res3]
    args(0)=[]VN
    ρ=𝜂SN
    res1(1)=λOBN
    res3(3)=λOBN
    φ=λOBN
    res2(2)=λOBN
  }SN
  φ=λOBN
}
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:84)
        at org.eolang.phi.AtLambda.get(AtLambda.java:63)
        at org.eolang.phi.AtBound.get(AtBound.java:56)
        at org.eolang.phi.AtNamed.get(AtNamed.java:66)
        at org.eolang.phi.PhDefault.attr(PhDefault.java:143)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.Dataized.take(Dataized.java:54)
        at org.eolang.EOseq.lambda$new$0(EOseq.java:55)
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:82)
        ... 13 more
Caused by: java.lang.IllegalArgumentException: org.eolang.phi.Attr$Exception: Error at sandbox.EOempty#x in sandbox.EOempty#193064360:{
  _order=[x]
  x(0)=𝜂SFN
  ρ=sandbox.EOapp#812265671:{
    _order=[args, res1, res2, res3]
    args(0)=[]VN
    ρ=𝜂SN
    res1(1)=λOBN
    res3(3)=λOBN
    φ=λOBN
    res2(2)=λOBN
  }SN
  φ=λOBN
}
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:84)
        at org.eolang.phi.AtLambda.get(AtLambda.java:63)
        at org.eolang.phi.AtBound.get(AtBound.java:56)
        at org.eolang.phi.AtNamed.get(AtNamed.java:66)
        at org.eolang.phi.PhDefault.attr(PhDefault.java:143)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.PhOnce.attr(PhOnce.java:65)
        at org.eolang.phi.Dataized.take(Dataized.java:54)
        at org.eolang.phi.Dataized.take(Dataized.java:72)
        at org.eolang.io.EOstdout.lambda$new$0(EOstdout.java:49)
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:82)
        ... 22 more
Caused by: org.eolang.phi.Attr$Exception: Error at sandbox.EOempty#x in sandbox.EOempty#193064360:{
  _order=[x]
  x(0)=𝜂SFN
  ρ=sandbox.EOapp#812265671:{
    _order=[args, res1, res2, res3]
    args(0)=[]VN
    ρ=𝜂SN
    res1(1)=λOBN
    res3(3)=λOBN
    φ=λOBN
    res2(2)=λOBN
  }SN
  φ=λOBN
}
        at org.eolang.phi.AtNamed.put(AtNamed.java:81)
        at org.eolang.phi.PhWith.lambda$new$1(PhWith.java:60)
        at org.eolang.phi.Data$Once.take(Data.java:65)
        at org.eolang.phi.PhOnce.toString(PhOnce.java:50)
        at org.eolang.phi.PhOnce.toString(PhOnce.java:50)
        at java.util.Formatter$FormatSpecifier.printString(Formatter.java:2886)
        at java.util.Formatter$FormatSpecifier.print(Formatter.java:2763)
        at java.util.Formatter.format(Formatter.java:2520)
        at java.util.Formatter.format(Formatter.java:2455)
        at java.lang.String.format(String.java:2940)
        at org.eolang.phi.Dataized.take(Dataized.java:59)
        at org.eolang.txt.EOsprintf.lambda$new$0(EOsprintf.java:58)
        at org.eolang.phi.AtLambda.lambda$toData$0(AtLambda.java:82)
        ... 33 more
Caused by: org.eolang.phi.Attr$Exception: This free attribute is already set, can't reset
        at org.eolang.phi.AtFree.put(AtFree.java:79)
        at org.eolang.phi.AtNamed.put(AtNamed.java:78)
        ... 45 more
ERROR: 1
</details>
